### ISSUE TITLE
Rename `show-whitespaces` to `show-whitespace`

### ISSUE THREAD

**Issue by alexherbo2** [mawww/kakoune#4624]
Reference:

- [`trim_trailing_whitespace`]
- [`editor.renderWhitespace`]

[`trim_trailing_whitespace`]: https://editorconfig.org#:~:text=trim_trailing_whitespace
[`editor.renderWhitespace`]: https://code.visualstudio.com/docs/getstarted/settings#:~:text=editor.renderWhitespace

**Comment by krobelus** [comment]
I don't think it's worth a breaking change but maybe accept both and only document/complete the new name?

**Comment by alexherbo2** [comment]
Isn’t whitespaces a typo?

**Comment by krobelus** [comment]
yeah, reminds me of [creat](https://manpage.me/?q=creat) which was never fixed..

**Comment by krobelus** [comment]
yeah, reminds me of [creat](https://manpage.me/?q=creat) which was never fixed..
